napari/utils/events/_tests/test_evented_list.py

```diff
@@ -366,3 +366,17 @@ class B(A, EventedList):
     assert hasattr(lst, 'events')
     assert 'boom' in lst.events.emitters
     assert lst == [1, 2]
+
+
+def test_event_group_depr(capsys):
+    events = EmitterGroup(b=None, deprecated={"a": "b"})
+    assert events.b == events.a
+    captured = capsys.readouterr()
+    assert captured.err == "emitter a is deprecated, b provided instead\n"
```

Comment:
As mentioned below, rather than printing to stderr, we should use warnings, and use `with pytest.warns(FutureWarning):`, see here: https://docs.pytest.org/en/stable/warnings.html#warns